### PR TITLE
Env multiple urls

### LINKED ISSUE #62: change the env variable to it allows more than one preset frontend to make it easier to pick instead of type it in
Until we are moving the input field from top, we can have a dropdown which list out all the preset frontend urls from env and also save it in cookie for later use.

### PR DESCRIPTION
**Pull request by MAX-786**
Fixes #62 

this includes example .env, you can add multiple urls in that format.

Screenshot from 2024-06-22 09-20-43

### CODE REVIEW

**Review by djay — APPROVED** [review]

**Comment by djay** [comment]
Where does it document the env variable to use and the format?

**Comment by MAX-786** [comment]
> Where does it document the env variable to use and the format?

They are in the `env.example` in root directory, I'll document it at section - setting up local instance of hydra.

**Comment by djay** [comment]
Include it in the readme I think

**Comment by MAX-786** [comment]
Here, its updated in readme:

https://github.com/collective/volto-hydra/blob/8887c85a5410a438e5ae068aeb7805dd5a5ba958/README.md?plain=1#L61C1-L76C93

File: README.md [hunk 1]
```diff
@@ -60,16 +60,20 @@ To test against a local hydra instance
     ```bash
     make start
     ```
+- You can also set your preset frontend URLs with environment variables, making them available in the admin UI. This allows you to switch between them seamlessly:
+    ```bash
+    RAZZLE_DEFAULT_IFRAME_URL=http://localhost:3002,https://hydra-blogsite-nextjs.vercel.app pnpm start
```
Comment (djay) [inline README.md:65]: whats the "pnpm start" for? mistake?

Comment (MAX-786) [reply README.md:65]: Its intended, if we want to pass env vars, it was discussed with Jeff

**Pull request merged**